src/GafferArnold/IECoreArnoldPreview/Renderer.cpp

```diff
@@ -479,8 +483,34 @@ class ArnoldAttributes : public IECoreScenePreview::Renderer::AttributesInterfac
 
 		};
 
+		struct Curves
+		{
+
+			Curves( const IECore::CompoundObject *attributes )
+			{
+				mode = attributeValue<string>( g_curvesModeAttributeName, attributes, "ribbon" );
+				minPixelWidth = attributeValue<float>( g_curvesMinPixelWidthAttributeName, attributes, 0.0f );
+			}
+
+			IECore::InternedString mode;
+			float minPixelWidth;
+
+			void hash( IECore::MurmurHash &h ) const
+			{
+				h.append( mode );
+				h.append( minPixelWidth );
+			}
+
+			void apply( AtNode *node ) const
+			{
+				AiNodeSetStr( node, "mode", mode.c_str() );
```

Comment:
OK, I've done the Cortex part in https://github.com/ImageEngine/cortex/pull/527, and updated this PR to work better with it, and added some talk of "oriented" mode to the documentation.